### DISCUSSION TITLE
Donation platform with Elementor & WP not 'activated'

### DISCUSSION THREAD

**Discussion by NmdakaDalai** [jonas-hoebenreich/wc-donation-platform#71] [Q&A] [answered]
How can I create a page that follows the same format as your demos? 
I have a woocommerce shop. I use Astra as the theme, Elementor as the page builder, and I have downloaded the donation platform to work with woocommerce.

I have created a loom video - (https://www.loom.com/share/160075eeebe4426ea3768015201a96cb?sid=3ae0a62f-efe7-4690-881c-024a61bab985) to show you my confusion and problem. If it is at all possible, would you be able to reply with a video showing how to do the correction? I find it hard to read and translate the directions, whereas kinetic learning is easier for me.

Many thanks

**Comment by jonas-hoebenreich** [comment, marked as answer]
I hope these visual guides help you:
How to fix "Donations not activated error":
https://scribehow.com/shared/Edit_Donation_Product_on_WordPress_Website__B-Hus_mXTl603e6qaWHUOA

How to recreate demo:
https://wcdp.jonh.eu/documentation/usage/recreate-one-of-the-demos/
https://scribehow.com/shared/Recreate_the_Demo__FoMx74mbTf6f_7dWimnOsQ

Best,
Jonas

**Comment by NmdakaDalai** [comment]
Thank you so much.
I discovered this after closing the page and walking away. Then reopening and looking at it in a different manner. I really appreciate your assistance.

Louise

Sent with [Proton Mail](https://proton.me/) secure email.
